store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -24,10 +24,14 @@ import com.dropbox.android.external.store4.StoreRequest
 import com.dropbox.android.external.store4.StoreResponse
 import com.dropbox.android.external.store4.StoreResponse.Data
 import com.dropbox.android.external.store4.StoreResponse.Loading
+import com.dropbox.android.external.store4.fresh
```

Comment:
you need to remove the TODO lines in `getAndFresh` test. i assume it will still pass but it was a pending TODO to be removed once this bug is fixed.

Reply:
there is 1 more.